pkg/controller/node/cidr_set.go

```diff
@@ -38,10 +38,16 @@ type cidrSet struct {
 func newCIDRSet(clusterCIDR *net.IPNet, subNetMaskSize int) *cidrSet {
 	clusterMask := clusterCIDR.Mask
 	clusterMaskSize, _ := clusterMask.Size()
-	maxCIDRs := 1 << uint32(subNetMaskSize-clusterMaskSize)
+
+	var maxCIDRs int
```

Comment:
The todo was added, as well as opened the bug for the bitmap. Any other comments?